packages/utils/src/lib/FpsTracker.ts

```diff
@@ -0,0 +1,36 @@
+/** @public */
+export class FpsTracker {
+	private startTime = 0
+	private name = ''
+	private frames = 0
+	private started = false
+	private frame: number | null = null
+
+	recordFrame = () => {
+		this.frames++
+		if (!this.started) return
+		this.frame = requestAnimationFrame(this.recordFrame)
+	}
+
+	start(name: string) {
+		this.name = name
+		this.frames = 0
+		this.started = true
+		if (this.frame !== null) cancelAnimationFrame(this.frame)
+		this.frame = requestAnimationFrame(this.recordFrame)
+		this.startTime = performance.now()
+	}
+
+	stop() {
+		this.started = false
+		if (this.frame !== null) cancelAnimationFrame(this.frame)
+		const duration = (performance.now() - this.startTime) / 1000
+		const fps = duration === 0 ? 0 : this.frames / duration
+		// eslint-disable-next-line no-console
+		console.log(`${this.name} FPS:`, fps)
```

Comment:
to be a little fancier :)
add bold/color, and put it in `console.debug`

```suggestion
		console.debug('%c${this.name} FPS%c: ${fps}', 'font-weight: bold; color: lightblue', 'font-weight: normal');
```

Reply:
Nice. Change it to blue, since it was hard to see on light background. But blue might be hard to see on dark backgrounds? 😂 

![CleanShot 2024-04-17 at 13 42 46](https://github.com/tldraw/tldraw/assets/2523721/f278ee00-f36f-4ae0-8a42-0507ad590513)